docs/payments-refunds/create-payments.md

```diff
@@ -145,6 +146,10 @@ curl https://api.mobilepay.dk/v1/payments/{PAYMENT_ID}/capture \
   }'
 ```
 
+:::danger
+If you receive 5xx response while capturing the payment most likely ended up in **captureStarted** state. You have to retry the capture in such case until you get either 2xx or 4xx. The capture call is idempotent.
```

Comment:
"...response while capturing the payment most likely..." looks a bit weird.
Maybe "...response while capturing, the payment most likely..." (not sure if correct) or
"...response while capturing the reservation most likely the payment..." (not sure if correct either)

Reply:
changed it. See if you like it

Reply:
Looks good!